Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -312,9 +318,12 @@ void IMixedRealityTouchHandler.OnTouchStarted(HandTrackingInputEventData eventDa
             }
 
             touchPoints.Add(eventData.Controller, eventData.InputData);
-            IsTouching = true;
 
-            PulseProximityLight(eventData);
+            // Make sure only one instance of this input source exists and is at the "top of the stack."
```

Comment:
Why does it need to be at the end of the list? Why not use a hashset?

Reply:
Using a set was my first reaction (since I wanted to avoid duplicates), but I also needed something sorted by "age." I thought about SortedSet too, but I didn't think the overhead warranted something this simple.